src/Microsoft.Health.Dicom.Functions/Duplicate/Models/DuplicateBatchArguments.cs

```diff
@@ -0,0 +1,51 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Health.Dicom.Core.Features.Model;
+
+namespace Microsoft.Health.Dicom.Functions.Duplicate.Models;
+
+/// <summary>
+///  Represents input to <see cref="DuplicateDurableFunction.DuplicateBatchAsync"/>
+/// </summary>
+public sealed class DuplicateBatchArguments
+{
+    /// <summary>
+    /// Gets or sets the number of threads available for each batch.
+    /// </summary>
+    public int ThreadCount { get; } = 5;
+
+    /// <summary>
+    /// Gets or sets the inclusive watermark range.
+    /// </summary>
+    public WatermarkRange WatermarkRange { get; }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="DuplicateBatchArguments"/> class with the specified values.
+    /// </summary>
+    /// <param name="watermarkRange">The inclusive watermark range.</param>
+    /// <param name="threadCount">The number of threads available for each batch.</param>    
+    /// <exception cref="ArgumentOutOfRangeException">
+    /// <paramref name="threadCount"/> is less than <c>1</c>.
```

Comment:
greater than or equal to?

Reply:
yeah, it's related to `ArgumentOutOfRangeException`, when <1, the exception will be thrown